src/app.js

```diff
@@ -1,12 +1,12 @@
 'use strict';
 
-const compression  = require('compression');
-const express      = require('express');
+const compression = require('compression');
```

Comment:
I line my requires up usually but have started to follow convention as popular JS projects don't do this. It's also not conventional and tedious to do in, say, `import` statements in React components which are typically numerous. The case where there is one long variable/package name makes it awkward to space everything else out that far, and where is the line drawn... do we align object properties too? I've accepted allowing syntax highlighting do its job for readability in this instance. These kinds of things are why I think [prettier](https://github.com/prettier/prettier) is a good idea :).